all_repos/github_api.py

```diff
@@ -44,12 +45,16 @@ def get_all(url: str, **kwargs: Any) -> List[Dict[str, Any]]:
     return ret
 
 
+def strip_trailing_dot_git(ssh_url: str) -> str:
+    return re.sub(r'\.git$', '', ssh_url)
```

Comment:
regex is overkill here:

```python
if ssh_url.endswith(':git'):
    return ssh_url[:-1 * len('.git')]
```

could you also underscore-prefix this helper and write a few short tests demonstrating its functionality?

(wish `str.removesuffix` had occurred sooner! but that's python3.9+)

Reply:
heh, i did think of `removesuffix` as well but presume this tool isn't 3.9+ just yet :p

Reply:
alternatively you could add to a utility module something like this:

```python
if sys.version_info >= (3, 9):
    str_removesuffix = str.removesuffix
else:
    def str_removesuffix(self: str, suffix: str) -> str:
        if self.endswith(suffix):
            return self[:-1 * len(suffix)]
        else:
            return self
```